src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber>{
+    private int number;
+
+    public static int LOTTO_MIN_NUMBER = 1;
+    public static int LOTTO_MAX_NUMBER = 45;
+
+    private LottoNumber(int number){
+        this.number = number;
+    }
+
+    public static LottoNumber makeNumberFromText(String text){
+        if(isBlank(text) || !isNumber(text))
+            throw new RuntimeException("숫자를 입력해주세요.");
+
+        return new LottoNumber(Integer.parseInt(text));
+    }
+
+    public static LottoNumber makeNumber(int number){
+        if(number < LOTTO_MIN_NUMBER || number > LOTTO_MAX_NUMBER)
+            throw new RuntimeException("범위에 해당하지 않는 수입니다.");
+
+        return new LottoNumber(number);
+    }
+
+    public final int getNumber(){
+        return number;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        LottoNumber that = (LottoNumber) o;
+        return number == that.number;
+    }
+
+    public boolean equals(int n){
+        return number == n;
+    }
+
+    public static boolean isBlank(String text){
+        return text.isEmpty() || text == null;
```

Comment:
```suggestion
        return text == null || text.isEmpty();
```
이 두 가지는 (논리적으로는) 같아 보이지만, 사실 서로 달라요
더 보기: [Short-circuit evaluation](https://en.wikipedia.org/wiki/Short-circuit_evaluation)